streamparse/util.py

```diff
@@ -267,7 +267,7 @@ def local_storm_version():
                                "\nSTDERR:\n{}".format(cmd, res.stdout,
                                                       res.stderr))
 
-    pattern = r'^Storm ([0-9.]+)'
+    pattern = r'Storm ([0-9.]+)'
     return parse_version(re.findall(pattern, res.stdout)[0])
```

Comment:
I actually thing the correct fix here would be to change this line to be:

```py
return parse_version(re.findall(pattern, res.stdout, flags=re.MULTILINE)[0])
```

since we really only want to match it at the beginning of a line.